internal/pgbackrest/pgbackrest_test.go

```diff
@@ -34,15 +39,19 @@ func TestStanzaCreateOrUpgrade(t *testing.T) {
 	ctx := context.Background()
 	configHash := "7f5d4d5bdc"
 	expectedCommand := []string{"bash", "-ceu", "--", `
-declare -r hash="$1" stanza="$2" message="$3" cmd="$4"
+declare -r hash="$1" stanza="$2" message="$3" cmd="$4" check_repo_cmd="$5"
 if [[ "$(< /etc/pgbackrest/conf.d/config-hash)" != "${hash}" ]]; then
     printf >&2 "%s" "${message}"; exit 1;
+elif ! bash -c "${check_repo_cmd}"; then
```

Comment:
:pick: Could we add a comment to explain why we are checking all the repo paths are in place?

Reply:
Will do.

Reply:
I put the comment on the source, rather than the test.